Frameworks/UIKit/UIButton.mm

```diff
@@ -243,6 +279,7 @@ - (void)_initUIButton {
 - (instancetype)initWithFrame:(CGRect)frame {
     if (self = [super initWithFrame:frame]) {
         [self _initUIButton];
+        self.font = [UIFont buttonFont];
```

Comment:
>self.font = [UIFont buttonFont]; [](start = 8, length = 32)

I know you removed the font setting on _titleLabel but why not just move self.font into _initUIButton instead of duplicating it in both initWithFrame/initWithFrame:xamlElement. It could just sit alongside self.adjustImage... near the top.